interfaces/builtin/mount_observe.go

```diff
@@ -49,11 +49,10 @@ const mountObserveConnectedPlugSecComp = `
 # restricted because it gives privileged read access to mount arguments and
 # should only be used with trusted apps.
 
-quotactl Q_GETQUOTA - - -
-quotactl Q_GETINFO - - -
-quotactl Q_GETFMT - - -
-quotactl Q_XGETQUOTA - - -
-quotactl Q_XGETQSTAT - - -
+# FIXME: restore detailed parameter filtering for quota control once
+# snap-confine can read this syntax.
+# See LP:#1662489 for context.
+quotactl
```

Comment:
NAK. quotactl was never granted before and this grants more privilege than mount-observe is meant to have.